docs/features/overview.md

```
@@ -1,57 +1,162 @@
-# Features
+# 🏛️ DeepTutor's Framework
 
-DeepTutor provides six specialized modules for AI-powered learning.
+<img src="/full-pipe.png" alt="DeepTutor Full-Stack Workflow" width="100%">
 
-<div class="feature-nav">
+### 💬 User Interface Layer
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)